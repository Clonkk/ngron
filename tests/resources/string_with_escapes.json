{
    "body": "tons of escapes should be ignored \r\n\r\n\" ok"
}