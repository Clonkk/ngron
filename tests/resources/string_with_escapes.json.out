Token(LeftBrace, '{')
Token(String, 'body')
Token(Colon, ':')
Token(String, 'tons of escapes should be ignored \r\n\r\n\" ok')
Token(RightBrace, '}')
Token(Eof, '')